_includes/index.md

```diff
@@ -1,60 +1,85 @@
 
-Don’t hesitate to take a look at or try out some of the early implementations of the draft spec!
+This work is performed in the [reactive-streams-jvm](https://github.com/reactive-streams/reactive-streams-jvm/) repository.
 
-For feedback on
+#### JavaScript Interfaces
 
-* the draft spec, the Reactive Streams initiative or general feedback:
+This working group defines a minimal set of object properties for observing a stream of elements within a JavaScript runtime environment. The goal is to provide a testable specification that allows different implementations to interoperate within that same runtime environment.
 
-    please open an Issue on the [Reactive Streams project](https://github.com/reactive-streams/reactive-streams/issues).
+This work is performed in the [reactive-streams-js](https://github.com/reactive-streams/reactive-streams-js/) repository.
 
-* the implementations themselves:
+#### Network Protocols
 
-    please direct that to the implementor via their preferred feedback system listed below.
+This working group defines network protocols for passing reactive streams over various transport media that involve serialization and deserialization of the data elements. Examples of such transports are TCP, UDP, HTTP and WebSockets.
 
-### Implementations of the draft spec
+This work is performed in the [reactive-streams-io](https://github.com/reactive-streams/reactive-streams-io/) repository.
 
-* Akka Streams
-   * See this [Activator template](http://www.typesafe.com/activator/template/akka-stream-scala) introducing the [Akka Project](http://akka.io/) implementation in Scala; a Java version will follow shortly.
+## Current State
+
+As of Feb 16, 2015 we have released version 1.0.0-RC3 of Reactive Streams for the JVM, including Java [API](http://www.reactive-streams.org/reactive-streams-1.0.0.RC3-javadoc), a textual [Specification](https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.0.RC3/README.md#specification) and a [TCK](http://www.reactive-streams.org/reactive-streams-tck-1.0.0.RC3-javadoc). Corresponding code artifacts are available on Maven Central:
+
+    <dependency>
+      <groupId>org.reactivestreams</groupId>
+      <artifactId>reactive-streams</artifactId>
+      <version>1.0.0.RC3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.reactivestreams</groupId>
+      <artifactId>reactive-streams-tck</artifactId>
+      <version>1.0.0.RC3</version>
+    </dependency>
+
+The source code for these is available on [github](https://github.com/reactive-streams/reactive-streams-jvm/tree/v1.0.0.RC3). Please use github issues for providing feedback.
+
+All artifacts and specifications are released under [Creative Commons Zero](http://creativecommons.org/publicdomain/zero/1.0) into the Public Domain.
+
+### Implementations (sorted alphabetically)
+
+#### On the JVM
```

Comment:
yes, good point; right now we have Reactor, RxJava, vert.x and Akka on RC4, right?